include/deal.II/fe/fe_system.h

```diff
@@ -510,11 +510,16 @@ class FESystem : public FiniteElement<dim,spacedim>
    */
   virtual std::string get_name () const;
 
-  // for documentation, see the FiniteElement base class
+  /**
+   * @copydoc FiniteElement<dim,spacedim>::clone()
+   */
```

Comment:
Why do you need this? doxygen automatically inherits the documentation from the virtual function in the base class. There is no need for the `@copydoc`.